flight/mcu/lib/teensyShared/rk4.cpp

```diff
@@ -0,0 +1,90 @@
+#include "rk4.h"
+
+#include <Arduino.h>
+
+#include <array>
+#include <cmath>
+
+using std::array;
+
+rk4::rk4() {
+    Atmosphere atmo;
+    atmo_ = atmo;
+}
+
+float rk4::cd(float alt, float vel) {
+    float mach = vel / (atmo_.get_speed_of_sound(alt));
+    // double cd = 0;
```

Comment:
delete commented out code (also in other places in this file). Also, are we using rk4 in flight code? I forget and I didn't see it anywhere